### DISCUSSION TITLE
How do I change the vertical tabs size?

### DISCUSSION THREAD

**Discussion by Leo-narda** [soulhotel/FF-ULTIMA#34] [Q&A]
This is probably super easy to do, but I'm not sure how do I change the tab size, or rather, the length, whenever it's open? I wanna use it without auto hide, so it always show, but make it a bit shorter so it takes a bit less space from the content itself.

**Comment by soulhotel** [comment]
The Modification page shows you how, its at the very top. (click the image for a better visual demonstration)
https://github.com/soulhotel/FF-ULTIMA/blob/main/doc/Modification.md

If you would like to specify an exact size, you would have to go into your chrome/theme/`all-global-positioning.css` file (line 8-28) and use your change the size options to your liking.

https://github.com/soulhotel/FF-ULTIMA/blob/00a599e2f9d68e9aec69c645e650a706c1200d8b/theme/all-global-positioning.css#L8-L28
